src/tests/test_opensubtitles_com.py

```
@@ -0,0 +1,614 @@
+)
+from program.services.post_processing.subtitles.providers.base import SubtitleItem
+
+
+@pytest.fixture
+def mock_config():
+    """Create a test configuration."""
+    return OpenSubtitlesComConfig(
+        enabled=True,
+        api_key="test_api_key_1234567890123456789012345",
+        username="testuser",
+        password="testpass",
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)